arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/common/CommonFragmentValues.java

```diff
@@ -60,18 +65,20 @@ public class CommonFragmentValues {
   @Setter public int numberOfNonStackRows;
   @Setter public boolean TLI;
   @Setter public int codeFragmentIndex = -1;
+  @Getter private TracedException tracedException;
```

Comment:
I think by default we should set it to `UNDEFINED` @lorenzogentile404. Do you not agree ?